src/App.tsx

```diff
@@ -1,24 +1,160 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
+import { createTodo, deleteTodo, getTodos } from './api/todos';
+import { Todo, TodoData } from './types/Todo';
+import { SelectedFiltering } from './types/SelectedType';
+import { TodoList } from './TodoMain/TodoList';
+import { TodoLoadingError } from './TodoLoadingError';
+import { AddTodoForm } from './AddTodo/AddTodoForm';
 
-const USER_ID = 0;
+const USER_ID = 10397;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [hasTitleError, setHasTitleError] = useState(false);
+  const [isLoadingError, setIsLoadingError] = useState(false);
+  const [filteringBy, setFilteringBy] = useState(SelectedFiltering.All);
+  const [tempTodo, setTempTodo] = useState<null | TodoData>(null);
+  const [isCreatingError, setIsCreatingError] = useState(false);
+
+  const loadTodos = useCallback(async () => {
+    setIsLoadingError(false);
+
+    try {
+      const todosFromServer: Todo[] = await getTodos(USER_ID);
+
+      setTodos(todosFromServer);
+    } catch (error) {
+      setIsLoadingError(true);
+    }
+  }, []);
+
+  const addTodo = useCallback(async (todoData: TodoData) => {
+    setIsCreatingError(false);
+    setTempTodo(todoData);
+
+    try {
+      await createTodo(todoData);
+      await loadTodos();
+      setTempTodo(null);
+    } catch {
+      setIsCreatingError(true);
+    }
+  }, []);
+
+  const removeTodo = useCallback(async (todoDeletedId: number) => {
+    const deletedTodo = await deleteTodo(todoDeletedId);
+
+    setTodos(prevTodos => prevTodos.filter(todo => todo !== deletedTodo));
+    loadTodos();
+  }, []);
+
```

Comment:
I see the deletion of one todo you have, but what about the "clear completed" button?